apps/emqx/src/emqx_listeners.erl

```diff
@@ -974,15 +978,21 @@ quic_listener_optional_settings() ->
         stateless_operation_expiration_ms
     ].
 
+-if(?EMQX_RELEASE_EDITION == ee).
 inject_root_fun(#{ssl_options := SslOpts} = Opts) ->
-    Opts#{ssl_options := emqx_tls_lib:opt_partial_chain(SslOpts)};
+    Opts#{ssl_options := emqx_auth_ext_tls_lib:opt_partial_chain(SslOpts)}.
```

Comment:
use persistent_term style injection to avoid static code dependency.
now apps/emqx depends on apps/emqx_auth_ext